### PR TITLE
add permission to android manifest for Android version below 11

### PR DESCRIPTION
**Pull request by dhouha123**
- Add the remaining permissions to the android manifest for Android version below 11

### CODE REVIEW

**Review by waelba — APPROVED** [review]

**Comment by waelba** [comment]
cc @DrissTM

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -17,6 +17,8 @@
     <!-- Gallery -->
     <uses-permission android:name="android.permission.READ_MEDIA_VIDEO" />
     <uses-permission android:name="android.permission.READ_MEDIA_IMAGES" />
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
```
Comment (DrissTM) [inline app/src/main/AndroidManifest.xml]:
```suggestion
    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" android:maxSdkVersion="28" />
```

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -17,6 +17,8 @@
     <!-- Gallery -->
     <uses-permission android:name="android.permission.READ_MEDIA_VIDEO" />
     <uses-permission android:name="android.permission.READ_MEDIA_IMAGES" />
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
```
Comment (DrissTM) [inline app/src/main/AndroidManifest.xml]:
```suggestion
    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" android:maxSdkVersion="32" />
```

**Review by DrissTM — CHANGES_REQUESTED** [review]
just some suggestion to scope the permission to relevant android versions

**Review by DrissTM — APPROVED** [review]
LGTM :+1:

**Pull request merged**